packages/app/test/cypress/integration/30-search/search.spec.ts

```diff
@@ -68,3 +68,183 @@ context('Access to legacy private pages', () => {
   });
 
 });
+
+context('Search all pages', () => {
+  const ssPrefix = 'search-all-pages-';
+
+  beforeEach(() => {
+    // login
+    cy.fixture("user-admin.json").then(user => {
+      cy.login(user.username, user.password);
+    });
+    // collapse sidebar
+    cy.collapseSidebar(true);
+  });
+
+  it(`Search all pages by word is successfully loaded`, () => {
+    const searchText = 'help';
+    cy.visit('/');
+    cy.get('.rbt-input').click();
+    cy.screenshot(`${ssPrefix}search-input-focused`, { capture: 'viewport'});
```

Comment:
please take a screenshot only the "search help" not as viewport
<img width="280" alt="Screen Shot 2022-06-13 at 19 47 01" src="https://user-images.githubusercontent.com/59536731/173337663-42e8dcc4-800f-41bf-bea5-7b1f6596e58c.png">